hadoop-ozone/recon/src/test/java/org/apache/hadoop/ozone/recon/api/TestNSSummaryEndpoint.java

```diff
@@ -299,13 +311,15 @@ public void testDiskUsage() throws Exception {
     Assert.assertEquals(KEY_SIX_SIZE, duDir4.getSize());
 
     // key level DU
-    Response keyResponse = nsSummaryEndpoint.getDiskUsage(KEY_PATH);
+    Response keyResponse = nsSummaryEndpoint.getDiskUsage(KEY_PATH,
+            false, false);
```

Comment:
Can we add tests for replica size? There are a few cases that need to be verified (key spanning into multiple blocks, container replica count < desired replica count etc)